src/qsub.jl

```diff
@@ -68,7 +70,7 @@ function launch(manager::Union{PBSManager, SGEManager}, params::Dict, instances_
 end
 
 function manage(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig, op::Symbol)
-    if op == :finalize
+    if op == :interrupt
```

Comment:
Why this change? Shouldn't `:interrupt` send a message that simulates a SIGINT? And `:finalize` should kill the workers.